pilot/pkg/model/policyattachment.go

```diff
@@ -0,0 +1,79 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"istio.io/api/type/v1beta1"
+	"istio.io/istio/pilot/pkg/features"
+	"istio.io/istio/pkg/config"
+	"istio.io/istio/pkg/config/constants"
+	"istio.io/istio/pkg/config/labels"
+	"istio.io/istio/pkg/config/schema/gvk"
+)
+
+type policyTargetGetter interface {
+	GetTargetRef() *v1beta1.PolicyTargetReference
+	GetSelector() *v1beta1.WorkloadSelector
+}
+
+type workloadSelectionOpts struct {
```

Comment:
Can we export this struct? I need this for a couple of tests outside of the model package. For example: [fuzz_test.go](https://github.com/istio/istio/pull/46560/files#diff-584f7975eb25111ba9a137c0362d4f7cd526bfcc814355af0281989e57724c37)

Reply:
I don't see why not